modules/kubernetes/grafana-agent/templates/operator-values.yaml.tpl

```diff
@@ -0,0 +1,9 @@
+resources:
+  requests:
```

Comment:
Should be able to set limit on the operator. I haven't see the memory usage of the operator above 77Mi so I guess something like 120 should be more than enough.